framework/include/mesh/MooseMesh.h

```diff
@@ -740,6 +740,11 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
    */
   std::vector<SubdomainID> getSubdomainIDs(const std::vector<SubdomainName> & subdomain_name) const;
 
+  /**
+   * Get the list of subdomains for the mesh (no active elements included)
```

Comment:
what do you mean by no active elements?

Reply:
So originally, within `SidesetAroundSubdomainUpdater`, when it called `MooseMeshUtils::hasSubdomainName(_mesh.getMesh(), name)` it wouldn't find the `subdomain = 9999` since it was added to the mesh, but had no elements associated with it.  From my understanding, when it go the mesh in this call, it looped over the elements, and grabbed the subdomain_ids.  This new call returns just the subdomains without the loop